content/en/tracing/llm_observability/sdk.md

```diff
@@ -265,19 +328,23 @@ The `LLMObs.annotate()` method accepts the following arguments:
 
 `input_data` 
 : optional - _JSON serializable type or list of dictionaries_ 
-<br />Either a JSON serializable type (for non-LLM spans) or a list of dictionaries with this format: `{"role": "...", "content": "..."}` (for LLM spans).
+<br />Either a JSON serializable type (for non-LLM spans) or a list of dictionaries with this format: `{"role": "...", "content": "..."}` (for LLM spans).  **Note**: Embedding spans are a special case and require a string, dictionary, or a list of dictionaries with this format: `{"text": "...", "name": "...", "score": float, "id": "..."}`.
```

Comment:
Do you mean **retrieval** here?
`**Note**: Embedding spans are a special case and require a string, dictionary, or a list of dictionaries with this format: `{"text": "...", "name": "...", "score": float, "id": "..."}`.`